styles/clientkit.css

```diff
@@ -0,0 +1,5 @@
+/* mixins */
```

Comment:
How should we pass the path to the clientkit/styles folder to clientkit-css, can this just be a config option?  Also if clientkit-css can't find the global mixins, it currently errors.  Should it instead just log the failure and carry on with loading the local mixins?